pxeboot.py

```diff
@@ -49,6 +49,12 @@ def parse_args() -> argparse.Namespace:
         nargs="+",
         help='Specify SSH public keys to add to the DPU\'s /root/.ssh/authorized_keys. Can be specified multiple times. If unspecified or set to "", include "/{host-path}/root/.ssh/id_ed25519.pub" (this file will be generated if it doesn\'t exist).',
     )
+    parser.add_argument(
```

Comment:
Would it be better to use RHEL subscription manager? This beaker code will only work within Red Hat network?

Reply:
that might be an interesting feature. Could be done.

However, note that CDA already has that feature (`postconfig: rh_subscription`), that lessens the need for it here.

Reply:
You can work on this in another PR. These changes are ok, but we need something that our partners can use.